src/api_manager/context/request_context.cc

```diff
@@ -279,6 +279,8 @@ void RequestContext::FillReportRequestInfo(
   info->url = request_->GetUnparsedRequestPath();
   info->method = GetRequestHTTPMethodWithOverride();
 
+  info->consumer_project_id = _consumer_project_id;
```

Comment:
is this duplicated?   I think info->check_response_info has the consumer_project_id, am I right?